tests/deployments/helm/helm_test.py

```diff
@@ -63,6 +63,12 @@ def helm_command_prefix(k8s_cluster):
     )
 
 
+def exec_helm(args):
+    proc = subprocess.run(args, shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Comment:
If you set `encoding='utf-8'` the encoding is done for stdout/stderr automatically. In 3.7 you can set `text=True` I think but we're on 3.6 afaik.